src/components/Buttons/Buttons.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export const Buttons = ({ reverseBtn, sortBtn, sortLengthBtn, resetBtn }) => (
+  <React.Fragment>
```

Comment:
используй короткую форму
```suggestion
  <>
```